src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx

```diff
@@ -93,6 +93,7 @@ export default createFragmentContainer(FairsRail, {
   fairs_module: graphql`
     fragment FairsRail_fairs_module on HomePageFairsModule {
       results {
+        internalID
```

Comment:
It’s better to use `id` for this, as Relay will already implicitly select it under the hood anyways and thus potentially saving some overfetching.

Reply:
Should the `internalID` selection still be added?